docs/usage.rst

```diff
@@ -219,6 +219,33 @@ Reporting on non-accounting data (queries)
 ------------------------------------------
 Check the module `mis_builder_demo <https://github.com/OCA/mis-builder/tree/10.0/mis_builder_demo>`_ to see how to create specific series of data for Committed purchases not yet invoiced.
 
+Example of query: we want a report showing the Total amount (HVAT) of all sales order confirmed during the current month and the previous month.
+
+Create a query:
+
+* Name: select a name (it must be a python expression, so no space in the name)
+* Model: look for the model on which you want to do the report. For example : Sale Order
+* Field to fetch: select in the list the field from the Model that will be used in the report. For example: Untaxed Amount
+* Fetched fields name: this will show the name of the field to use in the KPI expression later on. For example: amount_untaxed
+* Aggregate: choose between nothing, Sum, Max, Average or Min. Use either this aggregate field or you will need to add some aggregate expression in the KPI, because the query will give a list, not a number.
+* Date field: choose a date field available on the Model. For example: confirmation date of the sale order.
```

Comment:
Perhaps say a word explaining what this date field will be used for, i.e. selecting records that match the date period of the columns